cookbooks/bcpc-hadoop/attributes/hive.rb

```diff
@@ -1,3 +1,5 @@
-default[:bach][:hive][:metastore][:port] = 9083
-default[:bach][:hive][:hiveserver2][:port] = 10000
-default[:bcpc][:hive][:heap][:size]=1024
+default[:bcpc][:hadoop][:hive][:heap][:size]=1024
+default[:bcpc][:hadoop][:hive][:hiveserver2][:authentication][:method] = "KERBEROS"
+default[:bcpc][:hadoop][:hive][:hiveserver2][:authentication][:ldap_url] = ""
+default[:bcpc][:hadoop][:hive][:hiveserver2][:authentication][:ldap_domain] = ""
```

Comment:
Where are the values for ``ldap_url`` and ``ldap_domain`` stored for a test VM cluster? Do we need to install and configure OpenLDAP or something similar?